include/secp256k1.h

```diff
@@ -260,9 +260,26 @@ SECP256K1_WARN_UNUSED_RESULT int secp256k1_ec_pubkey_create(
   int compressed
 ) SECP256K1_ARG_NONNULL(1) SECP256K1_ARG_NONNULL(2) SECP256K1_ARG_NONNULL(3) SECP256K1_ARG_NONNULL(4);
 
+/** Compress a public key.
+ * In:     ctx:       pointer to a context object (cannot be NULL)
+ *         pubkeyin:  pointer to a 33-byte or 65-byte public key (cannot be NULL)
+ * In/Out: pubkeyout: pointer to a 33-byte array to put the compressed public key (cannot be NULL)
```

Comment:
You're right. Could still be the same for decompress as well, I'll take a look now.